client/bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/api/SubmissionRequestClient.kt

```diff
@@ -16,7 +16,7 @@ class SubmissionRequestClient(
         version: Int,
     ): RequestStatus =
         client
-            .post()
+            .get()
```

Comment:
looks like we need an itest for this one. I had created https://www.pivotaltracker.com/story/show/188165943

Reply:
I just added the itest as part of this ticket 👍